Java/PoolAndResourceFile/src/main/java/PoolAndResourceFile.java

```diff
@@ -0,0 +1,299 @@
+import java.io.*;
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+import java.time.Duration;
+import java.util.*;
+import java.util.concurrent.TimeoutException;
+
+import com.microsoft.azure.storage.*;
+import com.microsoft.azure.storage.blob.*;
+import org.apache.commons.io.IOUtils;
+
+import com.microsoft.azure.batch.*;
+import com.microsoft.azure.batch.auth.BatchSharedKeyCredentials;
+import com.microsoft.azure.batch.protocol.models.*;
+
+public class PoolAndResourceFile {
+
+    // Create IaaS pool if pool isn't exist
+    private static CloudPool CreatePool(BatchClient client, String poolId) throws BatchErrorException, IllegalArgumentException, IOException, InterruptedException, TimeoutException {
+        // Pool only have 1 A1 VM
+        String osPublisher = "OpenLogic";
+        String osOffer = "CentOS";
+        String poolVMSize = "STANDARD_A1";
+        int poolVMCount = 1;
+        long POOL_STEADY_TIMEOUT = 5 * 60 * 1000;
```

Comment:
does Java not have a notion of `TimeSpan`?  If we're using Java 8 it seems there is `Interval` or `Period` to use instead of doing this?